homeassistant/helpers/template.py

```diff
@@ -2395,6 +2446,11 @@ def __init__(
         self.globals["bool"] = forgiving_boolean
         self.globals["version"] = version
         self.tests["is_number"] = is_number
```

Comment:
```suggestion
        self.tests["is_number"] = is_number
        self.tests["number"] = is_number
```

Maybe we should also deprecate the filter `is_number`